data/json/professions.json

```diff
@@ -5158,5 +5158,81 @@
                 "panties"
             ]
         }
+    },
+        {
+        "type": "profession",
+        "ident": "bio_sci",
+        "name": "Mad Bionics Scientist",
+        "description": "Before all this you studied bionics. You tried your experiments on animals and yourself.", 
+        "traits": [
+            "ANIMALDISCORD",
+            "PSYCHOPATH"
+        ],
+        "skills": [
+            {"level": 4, "name": "electronics"},
+            {"level": 3, "name": "mechanics"}
+        ],
+        "points": 4,
+        "CBMs": [
+            "bio_alarm",
+            "bio_geiger",
+            "bio_armor_head",
+            "bio_batteries"
+        ],                
+        "entries": [
+            { "item": "goggles_nv", "charges": 50}
+        ],
+        "items": {
+            "both":[
+               "glasses_reading",
+               "coat_lab",
+               "gloves_rubber",
+               "pants",
+               "socks",
+               "boots_rubber",
+               "id_science" 
```

Comment:
Number of spaces has to be 4 per indent, not 3.

Reply:
On what?

Reply:
Every indent has to be a multiple of 4. Here you have 15 spaces.
If you're using notepad++, you can increase/decrease indent with tab/shift+tab.